src/editors/containers/VideoGallery/index.jsx

```diff
@@ -1,22 +1,25 @@
 import React, { useEffect } from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 import { selectors } from '../../data/redux';
 import hooks from './hooks';
 import SelectionModal from '../../sharedComponents/SelectionModal';
 import { acceptedImgKeys } from './utils';
 import messages from './messages';
 import { RequestKeys } from '../../data/constants/requests';
 
-export const VideoGallery = ({
-  // redux
-  rawVideos,
-  isLoaded,
-  isFetchError,
-  isUploadError,
-}) => {
+export const VideoGallery = () => {
+  const rawVideos = useSelector(selectors.app.videos);
+  const isLoaded = useSelector(
+    (state) => selectors.requests.isFinished(state, { requestKey: RequestKeys.fetchVideos }),
+  );
+  const isFetchError = useSelector(
+    (state) => selectors.requests.isFailed(state, { requestKey: RequestKeys.fetchVideos }),
+  );
+  const isUploadError = useSelector(
+    (state) => selectors.requests.isFailed(state, { requestKey: RequestKeys.uploadVideo }),
+  );
```

Comment:
Is this pattern better than passing the selectors as a state to props?

Reply:
The older mechanism of passing them as props makes it harder to understand where the data is coming from since some params are magically filed in and others need to be provided. I find this more explicit. 

Ideally, I'd also prefer if we had selectors like `selectors.requests.fetchVideos` which return a status like 'in_progress', 'done' etc. and then functions called `isFinished` etc to resolve that. That would make the above calls:

`const isLoaded = isFinished(useSelector(selectors.requests.fetchVideos));` 

That seems a lot cleaner than the current status quo, however it was a bigger refactoring than I had time for.